Making Sense/Models/CarCRUD.cs

```diff
@@ -2,13 +2,14 @@
 using Newtonsoft.Json;
 using System;
 using System.Collections.Generic;
+using System.Configuration;
 using System.IO;
 
-namespace Making_Sense
+namespace Making_Sense.Models
 {
     public class CarCRUD : ICarCrud
     {
-        private static string _path = @"C:\Users\Acer\source\repos\Making Sense\Making Sense\Cars.json";
+        private string JsonFile = ConfigurationManager.AppSettings["pathFileCar"];
```

Comment:
Podremos declarar `JsonFile` de solo lectura? 
```suggestion
        private readonly string JsonFile = ConfigurationManager.AppSettings["pathFileCar"];
```

Reply:
Bien, entiendo que es correcto, para que no se modifique el string que contiene la ruta del archivo, es así?

Reply:
Exacto, es una ruta que no tiene porqué cambiar una vez asignada.

Reply:
Perfecto, gracias!